packages/react/src/Alert/Alert.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Alert as AlertIcon, Checkmark, Close } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import React, { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { Icon } from '../Icon'
+import { VisuallyHidden } from '../VisuallyHidden'
+
+export interface AlertProps extends PropsWithChildren<HTMLAttributes<HTMLDivElement>> {
+  title?: string
+  type?: 'error' | 'success'
+  closeable?: boolean
+  icon?: boolean
+  onClose?: () => void
+}
+
+interface AlertCloseProps extends HTMLAttributes<HTMLButtonElement> {
+  size?: 'level-5' | 'level-6'
+}
+
+const AlertClose = forwardRef(
+  ({ className, size, ...restProps }: AlertCloseProps, ref: ForwardedRef<HTMLButtonElement>) => (
+    <button {...restProps} ref={ref} className={clsx('amsterdam-alert__close', className)} title="close">
```

Comment:
We don't use the `title` html attribute, see [the docs on interactive elements](https://amsterdam.github.io/design-system/?path=/docs/docs-designrichtlijnen-interactieve-elementen--docs#hover).

```suggestion
    <button {...restProps} ref={ref} className={clsx('amsterdam-alert__close', className)}>
```

Reply:
It caused a accessibility error in storybook.